.github/workflows/key4hep.yml

```diff
@@ -87,13 +87,13 @@ jobs:
                   --outputFileBase=bbudsc_3evt_GaudiRec \
                   --detectorModel=${{ matrix.detector_model }}
 
-  # Reconstruction with the v11 model (FCCee) using EDM4hep inputs and outputs
-  gaudi-v11-reco:
+  # Reconstruction with the ILD@FCCee models using EDM4hep inputs and outputs
+  gaudi-ILDatFCCee-reco:
```

Comment:
```suggestion
  gaudi-ILDetFCCee-reco:
```
(?)

Reply:
I don't understand that. 'et' as in French for 'and'?

Reply:
Probably ILDet(ector)

Reply:
Yes, currently it's `Dat` which I interpreted as a type for `Det`.

Reply:
I think one is supposed to read ILD at FCCee...

Reply:
Aaaahhh, well that also makes sense. In that case I just misread that.